### ISSUE TITLE
README lists gen_sctp as covered by this library, but it doesn't look it's true.

### ISSUE THREAD

**Issue by andoriyu** [meh/elixir-socket#33]

**Comment by f0rmiga** [comment]
Are you sure you have the SCTP library installed? What is your OS? When you post an issue, more informations are appreciated.

**Comment by andoriyu** [comment]
Yes, I use FreeBSD 10.2. Reason I'm asking is because searching this repo for SCTP only yields one result in readme file.

**Comment by meh** [comment]
Yeah, it's not wrapped yet, I have a partial implementation locally but it's not ready yet.
